level1c4pps/vgac2pps_lib.py

```diff
@@ -96,58 +98,262 @@
+                "offset": 12.69,
+                "rule": np.array(scene["sunzenith"].values) < 80.,
+                "comment": "based on NASA SBAFS for VZA < 5 and SZA < 80",
+            },
+            "tb37_twilight": {
+                "VIIRS channel": "M12",
+                "lutning": 0.9711,
+                "offset": 6.77,
+                "rule": np.array((scene["sunzenith"].values >= 80) & (scene["sunzenith"].values < 89.)),
+                "comment": "The linear average of the SBAFs for t37_day and t37_night. 80<= SZA <89",
+            },
+            "tb11": {
+                "VIIRS channel": "M15",
+                "lutning": 1.002,
+                "offset": -0.4,
+                "comment": "based on nadir collocation data for SZA 0 -180",
+            },
+            "tb12": {
+                "VIIRS channel": "M16",
+                "lutning": 0.9934,
+                "offset": 1.52,
+                "comment": "based on nadir collocation data for SZA 0-180",
+            },
+        },
+        "v6": {
+            "r06": {
+                "VIIRS channel": "M05",
+                "lutning": 0.9393,
+                "offset": 0.57,
+                "comment": "Based on collocation data, VZA < 3 for SZA < 60",
+            },
+            "r09": {
+                "VIIRS channel": "M07",
+                "lutning": 0.9104,
+                "offset": -0.90,
+                "comment": "Based on collocation data, VZA < 3 for SZA < 60",
+            },
+            "tb37_night": {
+                "VIIRS channel": "M12",
+                "lutning": 0.9967,
+                "offset": 0.85,
+                "rule": np.array(scene["sunzenith"].values) >= 89.,
+                "comment": "Based on collocations data for SZA >= 89",
+            },
+            "tb37_day": {
+                "VIIRS channel": "M12",
+                "lutning": 0.9455,
+                "offset": 12.69,
+                "rule": np.array(scene["sunzenith"].values) < 80.,
+                "comment": "Based on collocations for SZA < 80 o and VZA < 5",
+            },
+            "tb37_twilight": {
+                "VIIRS channel": "M12",
+                "lutning": 0.9711,
+                "offset": 6.77,
+                "rule": np.array((scene["sunzenith"].values >= 80) & (scene["sunzenith"].values < 89.)),
+                "comment": "The linear average of the SBAFs for t37_day and t37_night. 80<= SZA <89",
+            },
+            "tb11": {
+                "VIIRS channel": "M15",
+                "lutning": 1.003,
+                "offset": -0.77,
+                "comment": "Based on collocation data for VZA < 3 and SZA 0 -180",
+            },
+            "tb12": {
+                "VIIRS channel": "M16",
+                "lutning": 1.002,
+                "offset": -0.69,
+                "comment": "Based on collocation data for VZA < 3 and SZA 0-180",
+            },
+        }
+    }
+
+    strlen = max(len(k) for k in SBAF[V].keys())
```

Comment:
this variable seems not needed to me, and just confusing to have it.

Reply:
Well, I use this variable to make sure the output of the SBAF equations is more readable on the fly. i.e., by aligning the `=` signs. I could replace this variable with a constant, len('t37_twilight'), which is the longest string

Reply:
I'm not a fan of adding "complex" code just to get a nice print. It is better to add a unit test if this is to test that the function does what it is supposed to do,  or ignore that the print out is not perfectly aligned.